Rachnera Message Bust.rb

```diff
@@ -364,7 +364,7 @@ def valid_context?
   end
 
   def character_name
-    Busty::character_from_face($game_message.face_name, $game_message.face_index)
+    Busty::character_from_face(bust_face_name, $game_message.face_index)
```

Comment:
Oh, you've removed all references to `character_name` from within `bust_face_name` to avoid the infinite loop.

Should work better that way indeed as, unless I'm mistaken, that means the anticrash code in case of missing busts and similar shenanigans should run properly now. 👍 

(probably should replace `$game_message.face_index` with `bust_face_index` for consistency sake; but that's like super nitpicky)

Reply:
Yeah, I recall this also being important to make sure that the correct bust config (offsets etc) was selected. Fair point about the index, I'll test that shortly and update.

Reply:
Done.